modules/programs/password-store.nix

```diff
@@ -24,15 +24,10 @@ in
     settings = mkOption rec {
       type = with types; attrsOf str;
       apply = lib.mergeAttrs default;
-      default = {
-        PASSWORD_STORE_DIR = "${config.xdg.dataHome}/password-store";
-      };
-      defaultText = literalExpression ''
```

Comment:
I would suggest putting this behind a state version guard to maintain backwards compatibility. I.e., the default (in textual form) would be

``` nix
      defaultText = literalExpression ''
        { }                                                       for state version ≥ 25.11
        { PASSWORD_STORE_DIR = "$XDG_DATA_HOME/password-store"; } for state version < 25.11
      '';
```

Note, this would also affect the

```nix
services.pass-secret-service.storePath = lib.mkDefault cfg.settings.PASSWORD_STORE_DIR;
```

line in this module.